cli/lib/kontena/client.rb

```diff
@@ -332,13 +332,10 @@ def request(http_method: :get, path:'/', body: nil, query: {}, headers: {}, resp
         retry if refresh_token
       end
       raise Kontena::Errors::StandardError.new(401, 'Unauthorized')
-    rescue Excon::Errors::NotFound
-      raise Kontena::Errors::StandardError.new(404, 'Not found')
-    rescue Excon::Errors::Forbidden
-      raise Kontena::Errors::StandardError.new(403, 'Access denied')
-    rescue
-      logger.debug "Request exception: #{$!} - #{$!.message}\n#{$!.backtrace.join("\n")}"
-      handle_error_response
+    rescue Excon::Errors::HTTPStatusError => error
+      logger.debug "Request #{error.request[:method].upcase} #{error.request[:path]}: #{error.response.status} #{error.response.reason_phrase}: #{error.response.body}"
+
```

Comment:
Only remaining reference to `Kontena::Client#last_response` is in `Kontena::Cli::Master::LoginCommand#get_authorization_url`, and that uses `client.request(expects: [...])` so AFAIK it shouldn't be affected by this change.